Code/Framework/AzToolsFramework/AzToolsFramework/ComponentMode/ComponentModeCollection.cpp

```diff
@@ -295,6 +301,12 @@ namespace AzToolsFramework
             // wish to modify state to indicate this (e.g. appearance, functionality etc.)
             m_viewportEditorModeTracker->DeactivateMode({ GetEntityContextId() }, ViewportEditorMode::Component);
 
+            // notify other systems ComponentMode has ended
+            EditorComponentModeNotificationBus::Event(
+                GetEntityContextId(),
+                &EditorComponentModeNotifications::ComponentModeEnded
+            );
```

Comment:
update: See comment above and `m_viewportEditorModeTracker->DeactivateMode({ GetEntityContextId() }, ViewportEditorMode::Component);` call